src/site/content/en/blog/optimize-long-tasks/index.md

```diff
@@ -349,9 +349,13 @@ This is a simplistic example of how `postTask()` can be used. It's possible to i
 [`postTask()` is not supported in all browsers](https://caniuse.com/mdn-api_scheduler_posttask). You can use feature detection to see if it's available, or consider using [a polyfill](https://www.npmjs.com/package/scheduler-polyfill).
 {% endAside %}
 
-### Built-in yield with continuation
+### Built-in yield with continuation with `scheduler.yield`
```

Comment:
Double “with” if can think of a better way of putting it.

Reply:
Ah yeah, good catch. Will fix it with an inline commit.